src/contracts/interfaces/IRewardsCoordinator.sol

```diff
@@ -19,7 +19,8 @@ interface IRewardsCoordinator {
 
     /// @notice Reward type
     enum RewardType {
-        DELEGATED_STAKE
+        DELEGATED_STAKE,
+        PERFORMANCE
```

Comment:
Shouldn't we only allow commissions to be set for valid reward types? cc @8sunyuan ?